tamr_client/instance/backup.py

```diff
@@ -0,0 +1,94 @@
+from tamr_client import response
+from tamr_client._types import Instance, Session, URL
+from tamr_client.exception import TamrClientException
+
+
+class AlreadyCanceled(TamrClientException):
+    """Raised when attempting to cancel a backup that has already failed or been canceled.
+    """
+
+    pass
+
+
+class NotFound(TamrClientException):
+    """Raised when referencing a backup that does not exist on the server.
+    """
+
+    pass
+
+
+def get(session: Session, instance: Instance) -> list:
```

Comment:
I think this name should indicate that it will return all backups.  Maybe `get_all`.  It might be worth comparing to how we get a dataset's attributes, which we intentional chose to define as `tc.dataset.attributes`.  The equivalent here would be `tc.instance.backups`.

Reply:
it would be `tc.instance.backup.backups()` which i am fine with

Reply:
I think it's preferable to take advantage of the namespace and avoid repetition.  Any function in the `backup` module, probably shouldn't have the word backup in its name.  My early suggestion would involve moving this function `backups` into `instance.py` but I definitely see the weirdness in separating it from the other functions.

Reply:
what about `tc.instance.backup.all()` and `tc.instance.backup.from_id()`

Reply:
That seems pretty reasonable to me 👍

Reply:
using `get_all()` instead to keep consistent with transformations.py